DependencyInjection/Compiler/NotificationRegisterMappingsPass.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\NotificationBundle\DependencyInjection\Compiler;
+
+use Symfony\Bridge\Doctrine\DependencyInjection\CompilerPass\RegisterMappingsPass;
+use Symfony\Component\DependencyInjection\Definition;
+
+/**
+ * @author Salma Khemiri <chakroun.salma@gmail.com>
+ */
+class NotificationRegisterMappingsPass extends RegisterMappingsPass
+{
+    public function __construct($driver, array $namespaces, array $managerParameters, $driverPattern, $enabledParameter = false)
+    {
+        parent::__construct($driver, $namespaces, $managerParameters, $driverPattern, $enabledParameter);
+    }
+
+    public static function createMongoDBMappingDriver($mappings)
```

Comment:
Simply, I called DoctrineMongoDBMappingsPass::createXmlMappingDriver instead of this class.